include/bluetooth/mesh/gen_onoff.h

```diff
@@ -37,7 +37,7 @@ extern "C" {
 struct bt_mesh_onoff_set {
 	/** State to set. */
 	bool on_off;
-	/** Transition parameters. */
+	/** Transition parameters, or NULL if no transition was specified. */
```

Comment:
Should Scheduler Server set this pointer to NULL if the scheduled transition time is equal to 0?
It will emulate SetUnack for any Generic OnOff server including that extends LC.

Reply:
This now is changed. Please see the new version.